src/m365/teams/commands/tab/tab-list.ts

```diff
@@ -26,7 +26,7 @@ class TeamsTabListCommand extends GraphCommand {
   }
 
   public defaultProperties(): string[] | undefined {
-    return ['id', 'displayName', 'teamsAppTabId'];
+    return ['id', 'displayName'];
```

Comment:
Let's readd `teamsAppTabId`, since this property will still exist for text output.